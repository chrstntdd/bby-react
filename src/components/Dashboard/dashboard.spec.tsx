import * as React from 'react';
import { shallow, mount } from 'enzyme';

import ConnectedDashboard, { Dashboard } from './';

describe('<Dashboard/>', () => {
  it('should render without crashing', () => {
    shallow(<Dashboard />);
  });
});
