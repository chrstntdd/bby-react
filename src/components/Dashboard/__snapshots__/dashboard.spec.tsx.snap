// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard/> render method should render the dashboard 1`] = `
<div
  id="dashboard-wrapper"
>
  <DashboardHeader />
  <div
    id="main-content-area"
  >
    <Component
      clearTable={[MockFunction]}
      formatTable={[MockFunction]}
      printTable={[MockFunction]}
      shuffleTable={[MockFunction]}
      syncToDatabase={
        [MockFunction] {
          "calls": Array [
            Array [],
          ],
        }
      }
    />
    <section
      id="main-table-area"
    >
      <withRouter(Connect(ReduxForm)) />
      <withRouter(Connect(ProductTable)) />
    </section>
  </div>
</div>
`;
